plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreViewBase.java

```diff
@@ -48,72 +48,57 @@
 /**
  * PostgreViewBase
  */
-public abstract class PostgreViewBase extends PostgreTableReal implements DBSView
-{
-    private String source;
+public abstract class PostgreViewBase extends PostgreTableReal implements DBSView {
+    private String viewQueryResult;
 
-    public PostgreViewBase(PostgreSchema catalog)
-    {
+    public PostgreViewBase(PostgreSchema catalog) {
         super(catalog);
     }
 
     public PostgreViewBase(
         PostgreSchema catalog,
-        ResultSet dbResult)
-    {
+        ResultSet dbResult) {
```

Comment:
```suggestion
        ResultSet dbResult
    ) {
```
Formatting was fine here